pkg/certifier/components/scorecard/scorecard.go

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecard
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/guacsec/guac/pkg/assembler"
+	"github.com/guacsec/guac/pkg/assembler/graphdb"
+	"github.com/guacsec/guac/pkg/certifier"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j/dbtype"
+)
+
+type scorecard struct {
```

Comment:
this can probably be renamed to something more generic (together with the package), maybe "sourceArtifacts", and source_artifact package similar to be consistent with [root_package](https://github.com/guacsec/guac/blob/main/pkg/certifier/components/root_package/)

Reply:
Renamed!